aviary/docs/theory_guide/propulsion.ipynb

```diff
@@ -10,8 +10,27 @@
     "\n",
     "Aviary propulsion can be thought of as having two levels: system-level propulsion and individual engines. The top-level propulsion subsystem is model-agnostic and exists to organize engines and sum relevant outputs into system-level totals that other subsystems can use. Modeling of each type of engine on the aircraft is handled with `EngineModels`.\n",
     "\n",
-    "This page details how the core propulsion subsystem works when using `EngineDecks`. External propulsion subsystems (when wrapped by an `EngineModel`) are treated the same way by Aviary, but the details of what is computed and where will differ by model.\n",
-    "\n",
+    "This page details how the core propulsion subsystem works when using `EngineDecks`. External propulsion subsystems (when wrapped by an `EngineModel`) are treated the same way by Aviary, but the details of what is computed and where will differ by model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Testing Cell\n",
+    "# ********NOTE******** should we be using EngineModels vs EngineModel consistently here?\n"
```

Comment:
For notes like this, do you want a resolution in this PR? Or is this for future work?

Reply:
It depends on how complicated the resolution is, but I was planning on doing that as part of this PR